CefSharp/AddressChangedEventArgs.cs

```diff
@@ -14,6 +14,11 @@ public class AddressChangedEventArgs : EventArgs
         public IBrowser Browser { get; set; }
         public string Address { get; private set; }
 
+        /// <summary>
+        /// Called when a frame's address has changed. 
+        /// </summary>
+        /// <param name="browser"></param>
+        /// <param name="address"></param>
```

Comment:
As you've mentioned, `the address`, or `address` is fine here, it's pretty self explanatory